Description: switch test from ava to tape
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2019-11-24

--- a/test.js
+++ b/test.js
@@ -1,5 +1,5 @@
-import test from 'ava';
-import camelCase from '.';
+const test = require('tape');
+const camelCase = require('.');
 
 test('camelCase', t => {
 	t.is(camelCase('foo'), 'foo');
@@ -64,6 +64,7 @@
 	t.is(camelCase('桑德在这里。'), '桑德在这里。');
 	t.is(camelCase('桑德在这里。'), '桑德在这里。');
 	t.is(camelCase('桑德_在这里。'), '桑德在这里。');
+	t.end();
 });
 
 test('camelCase with pascalCase option', t => {
@@ -128,6 +129,7 @@
 	t.is(camelCase('РОЗОВЫЙ_ПУШИСТЫЙ-ЕДИНОРОГИ', {pascalCase: true}), 'РозовыйПушистыйЕдинороги');
 	t.is(camelCase('桑德在这里。', {pascalCase: true}), '桑德在这里。');
 	t.is(camelCase('桑德_在这里。', {pascalCase: true}), '桑德在这里。');
+	t.end();
 });
 
 test('camelCase with preserveConsecutiveUppercase option', t => {
@@ -159,6 +161,7 @@
 	t.is(camelCase('РозовыйПушистыйFOOдинорогиf', {preserveConsecutiveUppercase: true}), 'розовыйПушистыйFOOдинорогиf');
 	t.is(camelCase('桑德在这里。', {preserveConsecutiveUppercase: true}), '桑德在这里。');
 	t.is(camelCase('桑德_在这里。', {preserveConsecutiveUppercase: true}), '桑德在这里。');
+	t.end();
 });
 
 test('camelCase with both pascalCase and preserveConsecutiveUppercase option', t => {
@@ -190,6 +193,7 @@
 	t.is(camelCase('pозовыйПушистыйFOOдинорогиf', {pascalCase: true, preserveConsecutiveUppercase: true}), 'PозовыйПушистыйFOOдинорогиf');
 	t.is(camelCase('桑德在这里。', {pascalCase: true, preserveConsecutiveUppercase: true}), '桑德在这里。');
 	t.is(camelCase('桑德_在这里。', {pascalCase: true, preserveConsecutiveUppercase: true}), '桑德在这里。');
+	t.end();
 });
 
 test('camelCase with locale option', t => {
@@ -201,10 +205,12 @@
 	t.is(camelCase('ipsum-dolor', {pascalCase: true, locale: 'en-EN'}), 'IpsumDolor');
 	t.is(camelCase('ipsum-dolor', {pascalCase: true, locale: ['tr', 'TR', 'tr-TR']}), 'İpsumDolor');
 	t.is(camelCase('ipsum-dolor', {pascalCase: true, locale: ['en-EN', 'en-GB']}), 'IpsumDolor');
+	t.end();
 });
 
 test('invalid input', t => {
 	t.throws(() => {
 		camelCase(1);
 	}, /Expected the input to be/);
+	t.end();
 });
